pkg/arvo/app/language-server.hoon

```diff
@@ -95,6 +103,23 @@
       ~
     ==
   --
+::
+++  json-response
+  |=  jon=json
+  ^-  (list move)
+  :_  ~
+  :*
+    ost.bow
+    %http-response
+    (json-response:app (json-to-octs jon))
+  ==
+::
+++  coup
+  |=  [=wire saw=(unit tang)]
+  ^-  (quip move _this)
+  :_  this
+  ~
```

Comment:
We can't really return the stack tracing in the editor here because there's no guarantee that we have a HTTP request to respond to. When I get around to adding full ford builds and typechecking errors and so on, I'll add a polling system in hoon-language-server for surfacing that kind of stuff.